test/unit/spec/EnrollEmail_spec.js

```diff
@@ -0,0 +1,182 @@
+/* eslint max-params: 0 */
+define([
+  'q',
+  'okta',
+  '@okta/okta-auth-js/jquery',
+  'util/Util',
+  'helpers/mocks/Util',
+  'helpers/dom/AuthContainer',
+  'helpers/dom/EnrollEmailForm',
+  'helpers/dom/EnrollActivateEmailForm',
+  'helpers/dom/Beacon',
+  'helpers/util/Expect',
+  'sandbox',
+  'LoginRouter',
+  'helpers/xhr/MFA_ENROLL_email',
+  'helpers/xhr/MFA_ENROLL_ACTIVATE_email',
+  'helpers/xhr/SUCCESS',
+], function (Q, Okta, OktaAuth, LoginUtil, Util, AuthContainer,
+  EnrollEmailForm, EnrollActivateEmailForm, Beacon, Expect, $sandbox, Router,
+  xhrEnrollEmail, xhrEnrollActivateEmail, xhrSUCCESS) {
+
+  var { $ } = Okta;
+  var itp = Expect.itp;
+
+  Expect.describe('EnrollEmail', function () {
+
+    function setup (resp) {
+      var setNextResponse = Util.mockAjax();
+      var baseUrl = 'http://localhost:3000';
+      var authClient = new OktaAuth({
+        url: baseUrl,
+        transformErrorXHR: LoginUtil.transformErrorXHR
+      });
+      var successSpy = jasmine.createSpy('successSpy');
+      var router = new Router({
+        el: $sandbox,
+        baseUrl: baseUrl,
+        authClient: authClient,
+        globalSuccessFn: successSpy,
+        'features.router': true
+      });
+      Util.registerRouter(router);
+      Util.mockRouterNavigate(router, true);
+      setNextResponse(resp);
+      return Util.mockIntrospectResponse(router)
+        .then(function () {
+          router.refreshAuthState('dummy-token');
+          return Expect.waitForEnrollChoices();
+        })
+        .then(function () {
+          router.enrollEmail();
+          return Expect.waitForEnrollEmail({
+            router: router,
+            authContainer: new AuthContainer($sandbox),
+            beacon: new Beacon($sandbox),
```

Comment:
I think the only test missing now is `displays the correct factorBeacon` similar to other factors _spec files.

Reply:
🤦‍♂